content/events/subscribe-to-events/developer-guides/setup-subscription/_index.en.md

```diff
@@ -39,10 +39,22 @@ if your subscription request is not being accepted.
 
 ### Required subscription request properties
```

Comment:
`source` is still required, but I assume this will change for generic events. Should also mention using `resource` as a filter, since this now is referred to in content/events/_index.en.md

Reply:
Yeah, please do mention resource as well under the optional filters as it is not required for app subscriptions just yet. 
We will also go through the documentation and make sure it is up to date once the full support for resources in in place, so if you're feeling lazy we will handle it later